docs/cdn/asset-compression.md

```diff
@@ -0,0 +1,35 @@
+# Asset Compression Strategy
+
+### What and why compress assets?
+
```

Comment:
Levels are wrong here (should be `##`), but suggest removing this title entirely and combining with opening statement.

Reply:
Cool. Updated.